include/deal.II/multigrid/mg_tools.h

```diff
@@ -290,6 +290,12 @@ namespace MGTools
    * globally refined meshes). Values greater than 1 estimate the slowdown
    * one should see in a geometric multigrid v-cycle as compared with the same
    * computation on a perfectly distributed mesh hierarchy.
+   *
+   * This function is a collective MPI call between all ranks of the
+   * Triangulation and therefore needs to be called from all ranks.
+   *
+   * @note The value returned here is undefined if the
+   * construct_multigrid_hierarchy flag of the Triangulation is not set.
```

Comment:
Why not assert this? If the value doesn't make any sense, we shouldn't return anything...

Reply:
okay, that is more work. Will do after #9915.